### ISSUE TITLE
feat: expose load from csv file via Python driver

### ISSUE THREAD

**Issue by rad-pat** [databendlabs/bendsql#504]
Currently, only exposed is to load a data array via `stream_load` method on Python driver which involves reading the data from file and passing into the method
It would be good to have a method to load from CSV file with options, and/or the ability to stage a file with the driver bindings.

**Comment by everpcpc** [comment]
How about adding support to `stream_load` a pandas `DataFrame`?

**Comment by everpcpc** [comment]
If you are just seeking to load files, then you may try out this:
https://github.com/databendlabs/bendsql/blob/main/cli/tests/http/01-load_stdin.sh#L11

**Comment by rad-pat** [comment]
Yep, that's good for a test. How can I replicate with Python driver, passing the file name?

**Comment by everpcpc** [comment]
We will add support for python driver in next release.

**Comment by everpcpc** [comment]
Hi, we've released `v0.24.7`

**Comment by rad-pat** [comment]
Thanks, I will try this out

**Comment by rad-pat** [comment]
This is working, thanks 🙏
